accessible/toolboxview.component.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * AccessibleBlockly
+ *
+ * Copyright 2016 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Angular2 Component that details how a toolbox is rendered
+ * in AccessibleBlockly. Also handles any interactions with the toolbox.
+ * @author madeeha@google.com (Madeeha Ghori)
+ */
+blocklyApp.ToolboxView = ng.core
+  .Component({
+    selector: 'toolbox-view',
+    template: `
+      <h3 #toolboxTitle id="blockly-toolbox-title">Toolbox</h3>
```

Comment:
No, it's "Madeeha is afraid to break up html mid-tag in case it blows something up." I agree that it makes it hard to see important logic. I will experiment with breaking things up and putting things in tags in a certain order.